pkg/k8s/labels.go

```diff
@@ -65,6 +65,7 @@ const (
 )
 
 var proxyLabels = []string{
+	ControllerNSLabel,
```

Comment:
Since this label isn't formatted like the rest of the proxy labels, I think you'll need a special case for it in the `toOwnerLabel` func. Something like:

```
func toOwnerLabel(proxyLabel string) string {
	if proxyLabel == ControllerNSLabel {
		return "conduit_io_control_plane_ns"
	}
	stripped := strings.TrimPrefix(proxyLabel, "conduit.io/proxy-")
	if stripped == "job" {
		return "k8s_job"
	}
	return strings.Replace(stripped, "-", "_", -1)
}
```

Can you also update the test in `pkg/k8s/labels_test.go` to verify that this label is handled correctly?

Reply:
Oops, yes, thanks!